tests/cisco_ios/show_interface_link/cisco_ios_show_interface_link2.raw

```diff
@@ -0,0 +1,27 @@
+Port    Name               Down Time        Down Since
+Gi1/1                      37 weeks, 3 days, 5 hours, 52 minutes 18 secs      08:07:10  Mon Feb 14 2022
+Gi1/6                      15 weeks, 11 minutes 57 secs      13:47:31  Thu Jul 21 2022
+Gi1/10   Fitness           10 weeks, 5 days, 20 hours, 26 minutes 38 secs      17:32:50  Fri Aug 19 2022
```

Comment:
Is it possible for their to be spaces here? e.g. `ICE Port`

Reply:
Added test case and cleaned up some redundant lines.